### DISCUSSION TITLE
customisation statusCheck according to HTTP response

### DISCUSSION THREAD

**Discussion by louisonsarlinmagnus** [Lissy93/dashy#448] [Q&A] [answered]
In my Homelab I'm using Authelia as SSO.
Problem is that the `statusCheck` I get a red dot because of 401 return.
![image](https://user-images.githubusercontent.com/70757160/151366962-b6385455-d738-433d-981a-ea51dab21853.png)

Is there a way to add a "rule" saying 401=>Green ?

**Comment by lissy93** [comment]
This should be possible, I guess a new parameter would need to be passed to the status check server with accepted status codes, then if the request fails, the status code is looked at and if within accepted range it should return a pass. Seems doable 🤔 

I've opened a ticked for this in #456 :)

**Comment by lissy93** [comment, marked as answer]
Heya @louisonsarlinmagnus
I've implemented this in #451 and it's now released in 2.0.0 :) Lemme know what you think!
So you can use `statusCheckAcceptCodes` to specify allow codes, as outlined in the [Docs](https://github.com/Lissy93/dashy/blob/master/docs/status-indicators.md#allowing-alternative-status-codes).
If you're enjoying Dashy, please consider Staring the repo, or [Contributing to the project](https://github.com/Lissy93/dashy/blob/master/docs/contributing.md) :)

**Comment by louisonsarlinmagnus** [comment]
Thanks a lot for all the work you've done 😍  !!!
I'll test this asap !!
